src/ObjectUtils/CoffFile.cpp

```diff
@@ -120,6 +122,31 @@ uint64_t CoffFileImpl::GetExecutableSegmentOffset() const {
 bool CoffFileImpl::IsElf() const { return false; }
 bool CoffFileImpl::IsCoff() const { return true; }
 
+std::optional<PdbDebugInfo> CoffFileImpl::GetDebugPdbInfo() const {
+  const llvm::codeview::DebugInfo* debug_info = nullptr;
+  llvm::StringRef pdb_file_path;
+
+  // Only support PDB 70 until we learn otherwise.
+  constexpr uint32_t kPdb70Signature = 0x53445352;
+
+  // If 'this' was successfully created with CreateCoffFile, 'object_file_' cannot be nullptr.
+  CHECK(object_file_ != nullptr);
+  if (!object_file_->getDebugPDBInfo(debug_info, pdb_file_path) && debug_info != nullptr) {
```

Comment:
reverse the if to use early return? Also, we need to consume the llvm::Error, otherwise it will abort().